### ISSUE TITLE
Cannot approach the accuracy ~83% on cora

### ISSUE THREAD

**Issue by stridence** [danielegrattarola/keras-gat#26]
Hi danielegrattarola，great works!
But the classification accuracy for the transductive task on dataset cora is always approximately 80%~81%. I scrutimize the source code and keep the (hyper-)parameters consistent within the original paper except the epochs =200 and es_patient = 10 for the comparision with Kipf's GCN, however the reported 83% accuracy never appear. Any suggestion there or anything I missed? Thanks a lot in advance.

**Comment by danielegrattarola** [comment]
Hi, 

the number of epochs you are using is too low for achieving a good accuracy, you should keep the hyperparameters as they are if you want to reproduce the original results.

All the best, 
Daniele

**Comment by stridence** [comment]
It's working, thanks.
